easybuild/easyconfigs/i/imake/imake-1.0.8-GCCcore-10.2.0.eb

```
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'imake'
+version = '1.0.8'
+
+homepage = 'https://www.x.org/'
+description = """imake is a Makefile-generator that is intended to make it
+easier to develop software portably for multiple systems."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+builddependencies = [('binutils', '2.35', '', ('GCCcore', '10.2.0'))]
```

Please specify it like this instead;
```suggestion
builddependencies = [('binutils', '2.35')]
```

Can I also ask you to move the deps below the checksums line (just to be more consistent with other easyconfigs)